examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,140 @@
+# flake8: noqa
+"""
+========================================
+Release Highlights for scikit-learn 0.24
+========================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 0.24! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. **For an exhaustive list of
+all the changes**, please refer to the :ref:`release notes <changes_0_24>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+"""
+
+##############################################################################
+# Searching parameter space with successive halving
+# -------------------------------------------------
+# Successive halving, a state of the art method, is now available to
+# explore the space of the parameters and identify their best combination.
+# :class:`~sklearn.model_selection.HalvingGridSearchCV` and
+# :class:`~sklearn.model_selection.HalvingRandomSearchCV` can be
+# used as drop-in replacement for
+# :class:`~sklearn.model_selection.GridSearchCV` and
+# :class:`~sklearn.model_selection.RandomizedSearchCV`.
+# Successive halving is an iterative selection process. The first iteration is
+# run on a subset of the input sample. Only some of the parameter candidates
+# are selected for the next iteration, allowing to increase the size of the
```

Comment:
The use of "allowing" is unsettling because it suggests that there's a cause to effect relationship, while there's not. Increasing the number of resources is not a consequence of decreasing the number of candidates, it's just what defines the process. There's no other constraint than "it's just the way it is".